### PR TITLE
Use jalali core

### PR DESCRIPTION
**Pull request by slashmili**
Due to licensing issue raised in #148 , I've decided to take out the LGPL code and release it under a different package called [jalali-core](https://github.com/slashmili/jalali-core).

I'm planning to publish this change as a major release.

### CODE REVIEW

File: jdatetime/__init__.py [hunk 1]
```diff
@@ -14,9 +14,7 @@
 except ImportError:
     from _thread import get_ident
 
-from jdatetime.jalali import (
-    GregorianToJalali, JalaliToGregorian, j_days_in_month,
-)
+from jalali_core import GregorianToJalali, JalaliToGregorian, j_days_in_month
```
Comment (hramezani) [inline jdatetime/__init__.py:17]:
```suggestion
from jalali_core import GregorianToJalali, JalaliToGregorian, j_days_in_month

```

File: jdatetime/__init__.py [hunk 2]
```diff
@@ -14,9 +14,8 @@
...
+
```
Comment (hramezani) [inline jdatetime/__init__.py:18]:
```suggestion
```

**Comment by hramezani** [comment]
I think you should change the version in `jdatetime/__init__.py` as well.

https://github.com/slashmili/python-jalali/pull/151/files#diff-21d7a505e67f899179598c343c9cfdacfbdb8233f4ec3f84168f28450d36dfccR19

**Comment by slashmili** [comment]
Thanks! fixed it.

**Review by hramezani — APPROVED** [review]

**Pull request merged**